packages/lodestar/src/chain/blocks/processor.ts

```diff
@@ -58,7 +58,8 @@ export class BlockProcessor implements IService {
 
   public async start(): Promise<void> {
     const abortSignal = this.controller.signal;
-    pipe(
+    // ### Todo: Unknown promise duration
+    await pipe(
```

Comment:
What are the consequences of awaiting this function in the node start sequence?

Reply:
You would break chain start function(it's waiting for this method to end which will never happen). This is endless loop (it will only end if stop method is called)